thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="anomalies-table__values">
+  {{record.current}}/{{record.baseline}}
+</p>
+<p class="anomalies-table__change anomalies-table__change--{{color-delta record.change}}">
```

Comment:
screenshot didn't show the colors. Is this working?

Reply:
@justYves Yeah, a lot of these classes are stubs, so I can figure them out later.